### DISCUSSION TITLE
Just FYI on dead link in readme

### DISCUSSION THREAD

**Discussion by brueckmann** [Cimbali/pympress#328] [General]
Hello,

just wanted to let you know, that the link to "[Google style](http://www.sphinx-doc.org/en/latest/ext/example_google.html), contains rst syntax" [under Contributing in the Readme](https://github.com/Cimbali/pympress?tab=readme-ov-file#contributing) is dead.

**Comment by Cimbali** [comment]
Thanks! Fixed.

**Comment by brueckmann** [comment]
You’re very welcome! Thanks for the very quick fix.

Am 22.11.2024 um 14:04 schrieb Cimbali ***@***.***>:



Thanks! Fixed.
